Gemfile

```diff
@@ -94,6 +94,8 @@ group(:ruby_shadow) do
   gem "ruby-shadow", platform: :ruby
 end
 
+gem "chefstyle", group: :test
```

Comment:
could also consider just doing a `gem install` in travis and then not using `bundle exec` if this winds up ultimately not working for some reason...